BuildResidentialHPXML/resources/geometry.rb

```diff
@@ -1816,6 +1816,22 @@ def self.create_single_family_attached(runner:,
       end
     end
 
+    # Adiabatic gable walls for cathedral ceilings
+    if attic_type == HPXML::AtticTypeConditioned
+      attic_space.surfaces.each do |surface|
```

Comment:
It looks like you simply copied code from above? Please don't duplicate code ([DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)), either refactor the code or use a method. In this case, it seems like you can remove lines 1804-1816 and then just update your code here to be:
```suggestion
    # Adiabatic gable walls for cathedral ceilings
    if [HPXML::AtticTypeVented, HPXML::AtticTypeUnvented, HPXML::AtticTypeConditioned].include?(attic_type)
      attic_space.surfaces.each do |surface|
```